src/test/java/streams/part1/exercise/Exercise1.java

```diff
@@ -62,7 +80,10 @@ public void findMinimalAgeOfEmployees() {
         List<Employee> employees = Example1.getEmployees();
 
         // TODO реализация
-        Integer minimalAge = null;
+        Integer minimalAge = employees.stream()
+                                      .map(Employee::getPerson)
+                                      .map(Person::getAge).min(Comparator.comparingInt(left -> left))
```

Comment:
U can use .mapToInt(Person::getAge).min() as ages of persons are int values anyway.